### ISSUE TITLE
Question about the reported results

### ISSUE THREAD

**Issue by taoyang1122** [clovaai/CutMix-PyTorch#21]
Hi, thanks for the great job. I have a quick question. In the paper, it says that the reported result is the best result in the whole training process. Is this reasonable? Because we are not supposed to choose model based on the test set. And actually, in real application scenerios, we do not know which model is the best because we do not have test labels. So I think the final epoch result is more reasonable than the best result. Can you give the final epoch performance of ResNet-50 on ImageNet? Thanks a lot.

**Comment by hellbell** [comment]
@taoyang1122 
Our answer is here https://github.com/clovaai/CutMix-PyTorch/issues/7#issuecomment-516710134.
In short, we verified CutMix outperforms other methods whether we measure the best accuracy or the last accuracy.
